src/pages/BitcoinDashboardV2/components/ConvertBitcoinV2/index.tsx

```
@@ -0,0 +1,263 @@
+import { Decimal } from '@cosmjs/math';
+import { OBTCContractAddress, OraiBtcSubnetChain, OraichainChain } from 'libs/nomic/models/ibc-chain';
+import { DeliverTxResponse, toBinary } from '@cosmjs/cosmwasm-stargate';
+import { isDeliverTxFailure } from '@cosmjs/stargate';
+import { displayToast, TToastType } from 'components/Toasts/Toast';
+import { getStorageKey, handleErrorTransaction } from 'helper';
+import { useDepositFeesBitcoinV2, useGetWithdrawlFeesBitcoin } from 'pages/Balance/helpers';
+import { useRelayerFeeToken } from 'hooks/useTokenFee';
+import { btcTokens, oraichainTokens } from 'config/bridgeTokens';
+import { PendingWithdraws } from '../PendingWithdraws';
+
+const ConvertBitcoinV2: React.FC<{}> = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)